template/package.json

```diff
@@ -56,6 +56,8 @@
     "@react-native-community/eslint-config": "^0.0.5",
     "babel-jest": "^24.9.0",
     "eslint": "^6.4.0",
+    "eslint-plugin-import": "^2.18.2",
+    "eslint-plugin-prettier": "^3.1.1",
```

Comment:
should import and prettier go into our eslint plugin?

Reply:
Looks like we just need to include `eslint-plugin-echobind` and that resolves it.